tests/test-surround-view.cpp

```diff
@@ -180,31 +180,38 @@ add_stream (SVStreams &streams, const char *stream_name, uint32_t width, uint32_
 }
 
 static void
-write_image (const SVStreams &ins, const SVStreams &outs, uint32_t stream_idx)
+write_image (const SVStreams &ins, const SVStreams &outs, uint32_t outs_idx, bool new_frame)
```

Comment:
a. outs_idx make function complicated. why not use outs[idx] as parameters directly?
b. new_frame doesn't make much sense. suggest separate write_image to write_out_image and write_in_image.